ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -173,6 +172,18 @@ private WritableMap createWebViewEvent(WebView webView, String url) {
       event.putBoolean("canGoForward", webView.canGoForward());
       return event;
     }
+
+    /**
+     * Enables WebView debugging
+     *
+     * (Needs to make sure we're running on the UI thread, else it will throw
+     * a RuntimeException)
+     */
+    private void enableDebugging() {
```

Comment:
This shouldn't have this check (silent failures are very hard to debug!) and it probably doesn't need to be extracted into a separate function esp without the check